### DISCUSSION TITLE
EXIF support?

### DISCUSSION THREAD

**Discussion by calvin-powers** [thumbsup/thumbsup#228] [Q&A] [answered]
Just discovered this tool and am looking forward to trying it. just curious, does the tool do anything with a file's exif data?

**Comment by rprieto** [comment, marked as answer]
Hi, yes Thumbsup parses and processes all metadata (EXIF, IPTC, QuickTime...). They are used for 2 main things:
- some info like dates and captions are displayed on the galleries. This depends on the themes.
- you can use them to sort photos into albums, such as grouping them by date, or even creating an album for each IPTC keyword.
